server/routes/api/users/schema.ts

```diff
@@ -42,3 +83,58 @@ export const UsersDeleteSchema = BaseSchema.extend({
 });
 
 export type UsersDeleteSchemaReq = z.infer<typeof UsersDeleteSchema>;
+
+export const UsersInfoSchema = BaseSchema.extend({
+  body: z.object({
+    id: z.string().uuid().optional(),
+  }),
+});
+
+export type UsersInfoReq = z.infer<typeof UsersInfoSchema>;
+
+export const UsersActivateSchema = BaseSchema.extend({
+  body: BaseIdSchema,
+});
+
+export type UsersActivateReq = z.infer<typeof UsersActivateSchema>;
+
+export const UsersPromoteSchema = BaseSchema.extend({
+  body: BaseIdSchema,
+});
+
+export type UsersPromoteReq = z.infer<typeof UsersPromoteSchema>;
+
+export const UsersDemoteSchema = BaseSchema.extend({
+  body: z.object({
+    id: z.string().uuid(),
+    to: z.nativeEnum(UserRole).default(UserRole.Member),
+  }),
+});
+
+export type UsersDemoteReq = z.infer<typeof UsersDemoteSchema>;
+
+export const UsersSuspendSchema = BaseSchema.extend({
+  body: BaseIdSchema,
+});
+
+export type UsersSuspendReq = z.infer<typeof UsersSuspendSchema>;
+
+export const UsersResendInviteSchema = BaseSchema.extend({
+  body: BaseIdSchema,
```

Comment:
This seems to be missing `code`. It appears being used in the `users.resendInvite` API.

Reply:
Sorry, mistook it for the other endpoint :+1: